src/mscorlib/src/System/Array.cs

```diff
@@ -26,14 +26,8 @@ namespace System {
     // IList<U> and IReadOnlyList<U>, where T : U dynamically.  See the SZArrayHelper class for details.
     [Serializable]
     [ComVisible(true)]
-    #if CONTRACTS_FULL
-    [ContractClass(typeof(ArrayContracts))]
-    #endif
     public abstract class Array : ICloneable, IList, IStructuralComparable, IStructuralEquatable 
     {
-        /// <internalonly/>
-        internal Array() {}  // Don't call this.  Internal for ArrayContracts type.
```

Comment:
It should be internal as before (for full framework - where the trimming does not apply). I have added it back, with the same comment as what is in CoreRT.